packages/ckeditor5-emoji/src/emojirepository.ts

```diff
@@ -29,6 +30,16 @@ const SKIN_TONE_MAP: Record<number, SkinToneId> = {
 
 const BASELINE_EMOJI_WIDTH = 24;
 
+/**
+ * Object storing emoji support level by the operating system.
+ * Some emojis that are perceived as supported, are compound-emoji from previous version. For those emojis
+ * there has to be performed an additional width check to verify system support.
+ */
```

Comment:
```suggestion
/**
 * A map representing an emoji and its release version.
 * It's used to identify a user's minimal supported emoji level. 
 */
```

Reply:
Fixed in [7343da3](https://github.com/ckeditor/ckeditor5/pull/17864/commits/7343da3b2734aaf3d93b6e1e8e455c9a2bf881df)